### PR TITLE
Various cosmetic improvements

### PR DESCRIPTION
**Pull request by alexreg**
Related to the larger effort of https://github.com/rust-lang/rust/pull/58036.

### CODE REVIEW

File: src/comment.rs [hunk 1]
```diff
@@ -1819,8 +1810,7 @@ mod test {
         assert_eq!("// test1\n/*\n a bare line!\n\n      another bare line!\n*/", comment);
     }
 
-    // This is probably intended to be a non-test fn, but it is not used. I'm
-    // keeping it around unless it helps us test stuff.
+    // This is probably intended to be a non-test fn, but it is not used. We should keep this around unless it helps us test stuff to remove it.
```
Comment (scampi) [inline src/comment.rs]: This is better on two lines.

**Comment by scampi** [comment]
There are some test failures:

```
Mismatch at tests/source/itemized-blocks/wrap.rs:52:
 /// All the parameters ***except for
 /// `from_theater`*** should be inserted as sent
 /// by the remote theater, i.e., as passed to
-/// [`Theater::send`] on the remote actor.
+/// [`Theater::send`] on the remote actor
 fn func1() {}
 
 /// All the parameters ***except for
test test::self_tests ... FAILED
test test::verify_check_works ... ok
test test::verify_config_test_names ... ok
test unit_tests::test_format_code_block ... ok
test unit_tests::test_format_code_block_fail ... ok
test unit_tests::test_format_snippet ... ok
test unit_tests::test_no_panic_on_format_snippet_and_format_code_block ... ok
test utils::test::test_remove_trailing_white_spaces ... ok
test utils::test::test_trim_left_preserve_layout ... ok
test test::system_tests ... FAILED
failures:
---- test::self_tests stdout ----
internal error: line formatted, but exceeded maximum width (maximum: 100 (see `max_width` option), found: 102)
  --> src/matches.rs:28
   |
28 |     /// `true` if the arm is the last one in match expression. Used to decide on whether we should add
   |                                                                                                     ^^
   note= set `error_on_unformatted = false` to suppress the warning against comments or string literals
internal error: line formatted, but exceeded maximum width (maximum: 100 (see `max_width` option), found: 144)
    --> src/comment.rs:1813
     |
1813 |     // This is probably intended to be a non-test fn, but it is not used. We should keep this around unless it helps us test stuff to remove it.
     |                                                                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     note= set `error_on_unformatted = false` to suppress the warning against comments or string literals
internal error: line formatted, but exceeded maximum width (maximum: 100 (see `max_width` option), found: 101)
   --> src/config/file_lines.rs:214
    |
214 |     /// Returns `true` if `self` includes all lines in all files. Otherwise runs `f` on all ranges in
    |                                                                                                     ^
    note= set `error_on_unformatted = false` to suppress the warning against comments or string literals
internal error: line formatted, but exceeded maximum width (maximum: 100 (see `max_width` option), found: 102)
  --> src/patterns.rs:22
   |
22 | /// Returns `true` if the given pattern is short. A short pattern is defined by the following grammar:
   |                                                                                                     ^^
   note= set `error_on_unformatted = false` to suppress the warning against comments or string literals
warning: rustfmt may have failed to format. See previous 4 errors.
Ran 5 self tests.
thread 'test::self_tests' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: 1 self tests failed', src/test/mod.rs:249:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
---- test::system_tests stdout ----
Ran 346 system tests.
thread 'test::system_tests' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: 1 system tests failed', src/test/mod.rs:109:5
```

File: src/comment.rs (hunk 1 above)
Comment (alexreg) [reply src/comment.rs]: Agreed.

**Comment by alexreg** [comment]
@scampi Thanks for the review. I think I've fixed them now, but we'll see soon...

File: tests/source/fn-simple.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // rustfmt-normalize_comments: true
 
-fn simple(/*pre-comment on a function!?*/ i: i32/*yes, it's possible!  */   
+fn simple(/*pre-comment on a function!?*/ i: i32/*yes, it's possible!  */
```
[the remark this answers is not in the document]
Comment (alexreg) [reply tests/source/fn-simple.rs]: Ah, sorry!

File: src/rustfmt_diff.rs [hunk 3]
```diff
@@ -184,11 +174,13 @@ where
...
 }
 
-/// Convert a Mismatch into a serialised form which just includes
+/// Converts a `Mismatch` into a serialized form, which just includes
 /// enough information to modify the original file.
 /// Each section starts with a line with three integers, space separated:
+///
 ///     lineno num_removed num_added
-/// followed by (num_added) lines of added text.  The line numbers are
+///
```
[the remark this answers is not in the document]
Comment (alexreg) [reply src/rustfmt_diff.rs]: Sure.

File: tests/source/fn-simple.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexreg) [reply tests/source/fn-simple.rs]: (I think my editor did it automatically.)

**Comment by topecongiro** [comment]
Thanks!

**Comment by alexreg** [comment]
No problem. Appreciate the quick review & merge.

**Pull request merged**